doc/themes/scikit-learn/layout.html

```diff
@@ -26,6 +26,11 @@
   {% endif %}
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <script src="{{ pathto('_static/js/bootstrap.min.js', 1)}}" type="text/javascript"></script>
+  <script>
+     VERSION_SUBDIR = (function(groups) {
+         return groups ? groups[1] : null;
+     })(location.href.match(/^https?:\/\/scikit-learn.org\/([^\/]+)/));
```

Comment:
Checked that the above javascript regexp works as expected with the following code (run [here](https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_loc_href)),

```js
<!DOCTYPE html>
<html>
<body>

<button onclick="myFunction()">Match</button>

<p id="demo"></p>

<script>

var url = "http://scikit-learn.org/0.19/modules/linear_model.html#lars-lasso"

function matchUrl(groups) {
          return groups ? groups[1] : null;
      };
      
function myFunction() {
    var x = matchUrl(url.match(/^https?:\/\/scikit-learn.org\/([^\/]+)/));
    document.getElementById("demo").innerHTML = x;
}
</script>

</body>
</html>
```